tests/cli/cli_test.go

```diff
@@ -127,6 +127,14 @@ func writeNetrc(token, uri string) (*os.File, error) {
 	return f, nil
 }
 
+func deleteNetrc() error {
```

Comment:
Yup, it's https://github.com/remind101/empire/blob/master/tests/cli/login_test.go#L10 and https://github.com/remind101/empire/blob/master/tests/cli/login_test.go#L50